litestar/plugins/structlog.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+from litestar.cli._utils import console
```

Comment:
Console is only available when `rich` is installed, not sure this should be used here.

Reply:
Isn't Rich included by default now?

Reply:
Oh. Yes. Forgot about that :grimacing: